vitest.config.ts

```diff
@@ -13,7 +13,7 @@ export default defineConfig({
 
   test: {
     globals: true,
-    environment: 'jsdom',
+    environment: 'happy-dom',
```

Comment:
Has to switch back to `happy-dom` because `jsdom` throws errors on `:focus-within`. `happy-dom` too doesn't support that selector, but it silently ignores 😅

Reply:
👍